tools/rawtools/rawtools.xml

```diff
@@ -29,7 +29,14 @@
         ## TODO: To remove this patch when dropping support for Galaxy 17_09, adding profile="18.01" to the tool.
         export HOME=\$cwd  &&
 
-        ln -s '$input_raw' input.raw &&
+        mkdir raws_folder &&
+        #for $input_raw in $input_raws:
+            #set $input_name = $input_raw.element_identifier.split('/')[-1].replace(".raw", "") + ".raw"
+            ln -s -f '${input_raw}' './raws_folder/${input_name}' &&
+            #set $encoded_id = $__app__.security.encode_id($input_raw.id)
```

Comment:
uha, why is this needed?

Reply:
Ups, there was some old code here for testing, I've removed it.